contracts/modules/VersionManager.sol

```diff
@@ -175,13 +175,32 @@ contract VersionManager is IVersionManager, IModule, BaseFeature, Owned {
         return (1, OwnerSignature.Required);
     }
 
+    /* ***************** Static Call Delegation ************************* */
+
+    /**
+     * @notice This method is used by the VersionManager's fallback (via an internal call) to determine whether
+     * the current transaction is a staticcall or not. The method succeeds if the current transaction is a static call, 
+     * and reverts otherwise. 
+     * @dev The use of an if/else allows to encapsulate the whole logic in a single function.
+     */
+    function verifyIsInStaticCall() public {
```

Comment:
Maybe renaming that method to `onlyStaticCall` would make its intention even clearer?

Reply:
We generally followed the naming convention of using:
- `isX()` for methods returning a boolean when X is true
- `verifyX()` for methods reverting when `isX()` returns false
- `onlyX()` for modifiers calling `verifyX()`

but I could simplify the name to `verifyStaticCall()` if clearer?